custom_components/solaredge_modbus_multi/number.py

```diff
@@ -177,15 +177,15 @@ def __init__(self, inverter, config_entry, coordinator):
 
     @property
     def unique_id(self) -> str:
-        return f"{self._platform.uid_base}_storedge_backup_reserved"
+        return f"{self._platform.uid_base}_storage_backup_reserve"
 
     @property
     def name(self) -> str:
-        return "Storedge Backup Reserved"
+        return "Backup Reserve"
```

Comment:
All but 2 of the entities are prepended with "Storage" I think this should have the same. 
`        return "Storage Backup Reserve"`